service.partymode.autostart/addon.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.partymode.autostart"
+       name="Partymode autostart"
+       version="0.9.2"
+       provider-name="muttley">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.favourites" version="8.1.1"/>
+    </requires>
+    <extension point="xbmc.service" library="PartymodeAutostart.py"/>
+    <extension point="xbmc.python.script" library="main.py">
+        <provides>executable</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">PartyMode Autostart</summary>
+        <description lang="en">
+This service simply executes party mode or favourites on Kodi startup or on screensaver.
+
+Install, enable it (if not already) and enjoy!
+
+If you want change any delays, screensaver and other options go in addon settings.
+        </description>
+        <disclaimer lang="en">The author is not responsible for anything. Please review the code if you do not trust.</disclaimer>
+        <license>GNU GENERAL PUBLIC LICENSE Version 3, 29 June 2007</license>
+        <forum>https://forum.kodi.tv/showthread.php?tid=203290</forum>
+        <website>https://github.com/elbowz/partymode-autostart-xbmc-service</website>
```

Comment:
This info is still correct? If you're taking over an abandoned add-on this should point to the new upstream location of the code

Reply:
I'm the original developer...